File: node/HttpPost.js
var request = require('request');
console.log('+ SID token: ' + process.env.MAIN_ACCOUNT_SID + " " + process.env.MAIN_AUTH_TOKEN);
var basicAuth = "Basic " + Buffer.from(process.env.MAIN_ACCOUNT_SID + ":" + process.env.MAIN_AUTH_TOKEN).toString("base64");
var theRequestUrl = "https://api.twilio.com/2010-04-01/Accounts/" + process.env.MAIN_ACCOUNT_SID + "/Messages." + "json";
//
theFrom = process.env.MAIN_PN_7002;
theTo = process.env.MY_PHONE_NUMBER;
var theFormData = {
    From: theFrom,
    To: theTo,
    Body: 'Twilio support 2'
};
console.log('+ URL request: ' + theRequestUrl + " : " + JSON.stringify(theFormData));
request({
    method: "POST",
    headers: {
        "Authorization": basicAuth,
        'content-type': 'application/x-www-form-urlencoded'
    },
    url: theRequestUrl,
    formData: theFormData
}, function (error, response, body) {
    console.log(body);
});

// eof

File: vb/httppost.vb
' HTTP POST request
Imports System
Class Example
  Overloads Shared Sub Main(ByVal args() As String)
      Dim AccountSid As String = Environment.GetEnvironmentVariable("MASTER_ACCOUNT_SID")
      Dim GatewayAPI As String
      Console.WriteLine("+ Hello there.")
      GatewayAPI = "https://api.twilio.com/2010-04-01/Accounts/" + AccountSid + "/SMS/Messages.json?"
      Console.WriteLine("+ GatewayAPI: " + GatewayAPI)
      ' ogateway = Server.CreateObject("MSXML2.XMLHTTP.6.0")
      ' With ogateway
      ' .Open("POST", GatewayAPI, False, accountSid, authToken)
      ' .setRequestHeader("Content-Type", "application/x-www-form-urlencoded")
      ' .Send("From=+1" & twiliocell & "&To=+1" & tophone & "&Body=" & msg)
      ' ResponseString = .responseText
  End Sub
End Class
' eof